framework/doc/content/source/interfaces/SetupInterface.md

```diff
@@ -43,6 +43,20 @@ complete list of execution flags is provided by MOOSE are listed in the "registe
 
 !listing framework/src/base/MooseApp.C start=MooseApp::registerExecFlags() end=void
 
+The default value of "execute_on" is *linear* for most of MOOSE objects with the exception of:
+
+- The auxiliary kernels have the default value of *linear* and *timestep_end*.
+- The postprocessors have default value of *timestep_end* and *transfer*.
```

Comment:
is transfer default?

Reply:
No, my mistake. The code there is just allowing users to use `transfer` but not changing the default.